app/services/translation_service/api/translations.rb

```diff
@@ -0,0 +1,66 @@
+module TranslationService::API
+
+  class Translations
+
+    TranslationStore = TranslationService::Store::Translation
+
+
+    ## GET /translations/:community_id/
+    def get(community_id, request_params = {})
+      begin
+        params = TranslationService::DataTypes::Translation
```

Comment:
I saw from earlier comments that @ovan suggested using DataTypes to validate the params. @ovan was this what you had in your mind?

I'm not completely confident with this solution. To my understanding throwing an exception and immediately catching it is considered as a bad practice. I've always heard the mantra that "exceptions are for exceptional situations", e.g. network failure, disk failure etc. Someone calling API with invalid params is not that exceptional.